_source/logzio_collections/_log-sources/elastic-container-service.md

```diff
@@ -16,56 +16,160 @@ contributors:
   - idohalevi
   - mirii1994
   - shalper
+  - nshishkin
 shipping-tags:
   - aws
 order: 440
 ---
+<!-- tabContainer:start -->
+<div class="branching-container">
 
-This integration uses Filebeat in a Docker container to forward logs from Amazon Elastic Container Service (ECS) to Logz.io.
+* [Overview](#overview)
+* [Manual deployment](#manual-deployment)
+* [Automated CloudFormation deployment](#automated-cloudformation-deployment)
+{:.branching-tabs}
 
-To use docker-collector-logs, you'll set environment variables when you run the container.
-The Docker logs directory and docker.sock are mounted to the container, allowing Filebeat to collect the logs and metadata.
+<!-- tab:start -->
+<div id="overview">
 
-{% include log-shipping/docker-collector-logs.md %}
+This integration uses Fluentd in a Docker container to forward logs from your Amazon Elastic Container Service (ECS) cluster to Logz.io.
 
+Note that this repo refers to an EC2-based cluster. For Fargate-based cluster see this.
 
-#### Deploy the Docker collector
+<!-- info-box-start:info -->
+This integration refers to an EC2-based cluster. For Fargate-based cluster see [our Fargate documentation](https://docs.logz.io/shipping/log-sources/fargate.html).
+{:.info-box.note}
+<!-- info-box-end -->
 
+</div>
+<!-- tab:end -->
 
-<div class="tasklist">
+<!-- tab:start -->
+<div id="manual-deployment">
+
+#### Manual deployment
 
```

Comment:
Can we add a note that this method is for AWS Classic Console?
In the New Console they don't have an option to configure via JSON, which we need.